src/azureStorageHelper.ts

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+import * as storage from 'azure-storage';
+import { FileService } from 'azure-storage';
+import * as path from 'path';
+
+const DIRECTORY_NAME = 'ansible-playbooks';
+
+export async function uploadFilesToAzureStorage(localFileName: string, storageAccountName: string, storageAccountKey: string, fileShareName: string): Promise<void> {
+    const client = storage.createFileService(storageAccountName, storageAccountKey);
+
+    try {
+        await createFileShare(client, fileShareName);
```

Comment:
Current implementation is very straight-forward with great readability.
A minor suggestion is you may use cascading/chaining promises to simplify code.

Reply:
fixed